test/data/Example_AMEI_PARIS.xml

```diff
@@ -22,7 +22,7 @@
     <Outputs>
         <Output name="PlantGrowth" description="Growth of plant in hight (rate)" datatype="float" min="0" max="5"  unit="mm" uri="some url"/>
     </Outputs>
-    <Algorithm language="Python-ext"><![CDATA[
+    <Algorithm language="python_ext" platform = "OpenAlea"><![CDATA[
```

Comment:
No. You may just use Python_ext without OpenAlea. There is nothing specific to OpenAlea in the code.